openassessment/assessment/test/test_staff.py

```diff
@@ -196,6 +200,10 @@ def test_create_assessment_does_not_block(self, after_type, after_assess):
         if after_type == 'staff':
             return
 
+        requirements = self.STEP_REQUIREMENTS
```

Comment:
These are the same requirements we were already using in the second workflow update below. Previously, we had switched the requirements from `self.STEP_REQUIREMENTS` (which is `{}`) on the first check to the actual requirements on the second check.

Modifying the requirements mid-workflow seemed to be illogical, and was causing a weird issue with done status at one point during development, so I fixed the test to use the same requirements throughout.